src/admin/test_new_semver_release.rs

```diff
@@ -0,0 +1,71 @@
+use crate::{
+    db,
+    schema::{dependencies, versions},
+};
+
+use clap::Clap;
+use diesel::prelude::*;
+
+#[derive(Clap, Copy, Clone, Debug)]
+#[clap(
+    name = "test-new-semver-release",
+    about = "Scan the database for all versions and version requirements to \
+        ensure they can be parsed by the latest `semver` crate."
+)]
+pub struct Opts {}
+
+pub fn run(_opts: Opts) {
+    use self::dependencies::dsl::*;
+    use self::versions::dsl::*;
+
+    let url = db::connection_url(&dotenv::var("READ_ONLY_REPLICA_URL").unwrap());
+    let conn = PgConnection::establish(&url).unwrap();
+    conn.transaction::<_, diesel::result::Error, _>(|| {
+        let vers = versions.select(num).load(&conn)?;
+        test_versions(&vers);
+
+        let deps = dependencies.select(req).load(&conn)?;
```

Comment:
I'm not entirely sure about loading all dependencies in a single query, but this is against the replica database and I expect memory usage to be somewhere around 100 MB so I think this is okay for our current scale.